legion/legion/model/client.py

```diff
@@ -49,14 +49,17 @@ class ModelClient:
     Model HTTP client
     """
 
-    def __init__(self, model_id, model_version, host=None, http_client=None, use_relative_url=False, timeout=None):
+    def __init__(self, model_id, model_version, token, host=None, http_client=None, use_relative_url=False,
+                 timeout=None):
         """
         Build client
 
         :param model_id: model id
         :type model_id: str
         :param model_version: model version
         :type model_version: str
+        :param token: API token value to use (optional)
```

Comment:
Optional or required? In model signature token is required. Also `(Optional)` should be in the beginning of line